Source/CronosPlayUnreal/Public/PlayCppSdkActor.h

```diff
@@ -208,7 +208,10 @@ class CRONOSPLAYUNREAL_API APlayCppSdkActor : public AActor {
   private:
     static ::com::crypto::game_sdk::WalletconnectClient *_coreClient;
 
-    static const APlayCppSdkActor *_sdk;
+    static APlayCppSdkActor *_sdk;
```

Comment:
why not const?

Reply:
Because we need to change the internal status: https://github.com/cronos-labs/play-unreal-plugin/pull/258/files#diff-097b9056119eecdfb8e1f150f586de6b3ed6ef617853a734a6f0d94006274fa1R638

Reply:
const will compilation failure.

Reply:
how does it work with concurrent updates? or there are never concurrent updates?

Reply:
It only updates when walletconnect status changes, aka, connecting, disconnected, connected, updated. One user one session, no concurrent updates.

Reply:
race conditions can occur, because sendevent is called non unreal thread.

Reply:
should use mutex or queue if modification is necessary

Reply:
how about modifying inside async task?

Reply:
also `sendevent` can be changed to other, because it's modifying state, not only send event

Reply:
As long as sendEvent is called by one thread, no concurrent occurs, at least it is only called in callback function at this moment.

But it is ok to modify it as a thread safe function. Maybe in another PR?

Reply:
See https://github.com/cronos-labs/play-unreal-plugin/issues/264

Reply:
Close #264, kept AsyncTask, AysncTask works but and changed the title for: https://github.com/cronos-labs/play-unreal-plugin/issues/214

Reply:
See https://github.com/cronos-labs/play-unreal-plugin/pull/258/files#diff-097b9056119eecdfb8e1f150f586de6b3ed6ef617853a734a6f0d94006274fa1R641 as well.